src/modules/fancyzones/lib/ZoneSet.cpp

```diff
@@ -162,41 +164,63 @@ IFACEMETHODIMP ZoneSet::AddZone(winrt::com_ptr<IZone> zone) noexcept
     return S_OK;
 }
 
-IFACEMETHODIMP_(winrt::com_ptr<IZone>)
-ZoneSet::ZoneFromPoint(POINT pt) noexcept
+IFACEMETHODIMP_(std::vector<int>)
+ZoneSet::ZonesFromPoint(POINT pt) noexcept
 {
-    winrt::com_ptr<IZone> smallestKnownZone = nullptr;
-    // To reduce redundant calculations, we will store the last known zones area.
-    int smallestKnownZoneArea = INT32_MAX;
-    for (auto iter = m_zones.rbegin(); iter != m_zones.rend(); iter++)
+    const int SENSITIVITY_RADIUS = 20;
```

Comment:
This PR also changes the current snapping behavior with this param introduction.
Example (red dot is a cursor position):
Before:
![image](https://user-images.githubusercontent.com/1828123/78576340-34a43d00-7835-11ea-9f33-09c4f0e7d343.png)
After:
![image](https://user-images.githubusercontent.com/1828123/78576646-9ebce200-7835-11ea-99fd-4804257d2059.png)


So basically we can snap windows to zones now from a "distance" of 20px. Since I assume that param was introduced for a more smooth UX when zones are overlapping, perhaps we could collect the zones with sensitivity, check if they overlap, and if they *don't*, we filter them again, now with `SENSITIVITY_RADIUS == 0`?

Reply:
I think we should fix this, since it may cause users to report it as a bug.

Reply:
Okay, I'll reconsider how zones are detected

Reply:
I pushed a commit which should address this, please take a look.